lessons/Early-Math/commutative-property/langs/en-us/index.adoc

```diff
@@ -0,0 +1,120 @@
+
+=== Investigate
+
+[.lesson-instruction]
+- Turn to @printable-exercise{comm-prop-table.adoc}.
+- How many different ways can you apply the Commutative Property? Draw as many different Circles of Evaluation as you can.
+
+This activity is similar to one students’ completed during the previous lesson. In both activities, students develop equivalent expressions. That said, in this lesson, students cannot write down _any_ equivalent circle; they must develop equivalent Circles that represent the commutativity.
+
+As students work, draw their attention to the operation as they rearrange the values inside the circle, helping them to notice that the Commutative Property does not apply for division and subtraction.
+
+[.lesson-instruction]
+- How come some of the Circles had fewer equivalent Circles than others?
+** _Possible response: When expressions included addition and multiplication, there were many equivalent expressions, whereas division and subtraction resulted in fewer (or no) equivalent expressions._
+- What did you notice about the final Circle on the worksheet?
+** _Possible response: It was impossible to apply the Commutative Property because there were no instances of multiplication or addition, only division and subtraction._
+
+=== Common Misconception
+
+Students may attempt to apply other properties when developing equivalent Circles. In such instances, help students to see that their Circle is indeed equivalent, is not an example of the Commutative Property at work.
+
+=== Synthesize
+
+- For which operations does the Commutative Property hold?
+** _The Commutative Property holds for multiplication and addition, but not division or subtraction._
+- When applying the Commutative Property, did the groupings of the numbers ever change?
+** _No; the Commutative Property does not allow for different groupings of numbers. When we apply the Commutative Property, we change the *order* of the values._
+- Which type of Circles had the most equivalent Circles? Which Circles had the least amount of equivalent Circles (or none!)?
+** _Possible response: Circles with only addition and multiplication, as well as Circles with many nested Circles, allowed for more equivalent representations. When Circles included division and subtraction, I could not develop as many equivalent expressions._
+
+== Tying it All Together
+
+=== Overview
+
+Students connect and apply what they have learned so far about translation, computation, and the Commutative Property.
+
+=== Launch
+
+Explain to your students that Claire and Walker are both translating expressions in words into Circles of Evaluation. Their teacher asks them to represent the following expression: "one half the product of five and six."
+
+[.embedded, cols="^.^1,^.^1", grid="none", stripes="none" frame="none"]
+|===
+
+| Claire draws this:				| Walker draws this:
+|@show{(coe  '(* 1/2 (* 5 6)))}		| @show{(coe  '(* (* 5 6) 1/2))}
+|===
+
```

Comment:
I'm not sure this is the most interesting discussion.

As a teacher interested in getting my students to fluency and precision with writing code, I saw the commutative property lead to many errors as most students who didn't stick to the order didn't do so because they were thinking about whether it was addition/multiplication or subtraction/division. They just either followed the order or they didn't. 

A much more important and interesting discussion I would want my students to debate is about the student who write the circle (/ (*5 6) 2), which I would see as an equally excellent (if not better) way of translating those words.